superset/models/dashboard.py

```diff
@@ -414,13 +415,12 @@ def export_dashboards(  # pylint: disable=too-many-locals
                 "native_filter_configuration", []
             )
             for native_filter in native_filter_configuration:
-                session = db.session()
```

Comment:
This session was never closed, though it's unclear why a new session was created.